cmd/diagnostic_test.go

```diff
@@ -90,16 +86,16 @@ token: mytoken
 				session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
 				Expect(err).ShouldNot(HaveOccurred())
 				Eventually(session.Err.Contents()).Should(BeEmpty())
-				Eventually(session.Out).Should(gbytes.Say("GraphQL API address: https://example.com/graphql"))
+				Eventually(session.Out).Should(gbytes.Say(
+					fmt.Sprintf("GraphQL API address: %s", testServer.URL())))
 				Eventually(session.Out).Should(gbytes.Say("OK, got a token."))
 				Eventually(session).Should(gexec.Exit(0))
 			})
 		})
 
-		Describe("token, host, and old endpoint set in config file", func() {
+		Describe("old endpoint set in config file", func() {
```

Comment:
I think this should be the opposite of what was deleted: "token, host set in config file"

Reply:
The description here is probably confusing, but the behavior we're specifying is:

* Have anything in host file
* Endpoint receives a fully-qualified address (via `--endpoint` above)
* The `--endpoint` address is reported as the API address in the diagnostic output

Reply:
Ahh, so the purpose of this test is to ensure the ordering is correct when both are set.  I'll add a small PR to capture this.

Reply:
@eric-hu It's documented here:
https://github.com/CircleCI-Public/circleci-cli/blob/d9f59f7b1f79fe202cfc5af1d1a6c41193b9557f/api/api.go#L169-L176

As well as additional test written for the behavior here:
https://github.com/CircleCI-Public/circleci-cli/blob/d9f59f7b1f79fe202cfc5af1d1a6c41193b9557f/api/api_test.go#L28-L32

(Not including the test I added in ccbe2d9)

Reply:
Thanks! I saw the `diagnostic_test.go` update from `ccbe2d9` and then updated my comment above since that seemed clear enough.  I'm guessing Github sent you an email notification for my original comment though?